client/my-sites/ads/main.jsx

```diff
@@ -247,8 +251,19 @@ class AdsMain extends Component {
 			component = this.renderInstantActivationToggle( component );
 		}
 
+		const adsProgramName = isJetpack ? 'Ads' : 'WordAds';
```

Comment:
I don't think we reuse the isJetpack bool, so potentially this can be moved to mapStateToProps

Reply:
Makes sense!